package.json

```diff
@@ -79,6 +79,7 @@
     "eslint-plugin-jest": "^27.9.0",
     "eslint-plugin-jsonc": "^2.14.1",
     "eslint-plugin-prettier": "^5.1.3",
+    "@jest/globals": "^29.7.0",
```

Comment:
The ordering here is incorrect. Can you move it to the top of `devDependencies` since `npm i -D` would normally add it in alphabetical order?

Reply:
Done